apps/web/src/services/my-page/my-account/user-account/modules/MultiFactorAuthentication.vue

```diff
@@ -28,58 +28,56 @@ const contextMenuItems = [
 
 const state = reactive({
     loading: false,
-    mfa: computed(() => store.state.user.mfa || undefined),
     userId: computed(() => store.state.user.userId),
     domainId: computed(() => store.state.domain.domainId),
-    isVerified: false,
+    //
+    mfa: computed(() => store.state.user.mfa || undefined),
+    isVerified: computed(() => state.mfa?.state === 'ENABLED'),
     enableMfa: false,
-    isModalVisible: false,
-
     // Currently, only email is supported.
     selectedItem: contextMenuItems[0].name,
 });
 
+const modalState = reactive({
+    isModalVisible: false,
+    modalType: '' as 'verify' | 'disabled' | 'change' | 'new',
```

Comment:
Question: doesn't modal need to have default value?

Reply:
It's okay even without a default value because it changes depending on the click context!